Automattic/woocommerce-payments
# Add fraud and risk tools discoverability banner to WooCommerce Core Home page

Fixes Automattic/woocommerce-payments-server#3135

#### Changes proposed in this Pull Request



This adds the Fraud and Risk tools banner to the WC Homescreen. It's using a slot that was added to WC Core on 7.4, so versions before that may not display this. According to [this look](https://looker.a8c.com/looks/2811) it's not significant.

h/t to @joshuatf and @rjchow for helping solving the slotfill issues (the error was caused by using the bundled version of `@wordpress\components`, using it from the window object, solved it instantly.)






#### Screenshots
![image](https://user-images.githubusercontent.com/3295/228734997-515b32be-4231-48b6-a622-41165b92c0d3.png)


![image](https://user-images.githubusercontent.com/3295/228735029-83ce4231-c088-43ce-a8d7-cbcb337a961d.png)





#### Testing instructions





* Checkout this branch.
* Enable the `wcpay_fraud_protection_settings_active` flag on your client
    * If you don't have it set yet, run: `npm run wp option add wcpay_fraud_protection_settings_active 1`
* Otherwise, update it by running: `npm run wp option update wcpay_fraud_protection_settings_active 1``
* Open _WooCommerce > Home_ page and check if you see the discoverability banner. 
* It should work as described in https://github.com/Automattic/woocommerce-payments/pull/5611 testing instructions, no need to test it again because it's rendering exactly the same component.
-------------------


**Post merge**
